src/separateLater/Sentinel1/Metadata/Sentinel1Metadata.jl

```diff
@@ -0,0 +1,749 @@
+include("MetadataUtil.jl")
+
+""""
+Structures and constructures for the metadata. 
+    MetaDataSentinel1
+        - ::Header
+        - ::ProductInformation
+        - ::ImageInformation
+        - ::SwathTiming
+        - ::BurstsInfo
+        - ::GeolocationGrid
+"""
+#currently, it is called "MetaDataSentinel1" to allow for a later abstract structures with MetaData for different satellites.
+
+
+
+##################
+### Structures
+##################
+##base.@kwdef not a part of stable julia. Probably will be future release(?)
+"""
+Header
+
+returns structure of Header from metadata in .xml
+"""
+Base.@kwdef struct Header
+    missionId::String
+    productType::String
+    polarisation::String
+    missionDataTakeId::Int
+    swath::Int
+    mode::String
+    startTime::DateTime
+    stopTime::DateTime
+    aqusitionTime::Millisecond
+    absoluteOrbitNumber::Int
+    imageNumber::String
+end
+
+
+"""
+ProductInformation
+
+returns structure of product information 
+"""
+Base.@kwdef struct ProductInformation
+    pass::String
+    timelinessCategory::String
+    platformHeading::Float64
+    projection::String
+    rangeSamplingRate::Float64
+    radarFrequency::Float64
+    azimuthSteeringRate::Float64
+end
+
+
+
+"""
+ImageInformation
+
+returns structure of ImageInformation from metadata in .xml
+"""
+Base.@kwdef struct ImageInformation
+    rangePixelSpacing::Float64
+    azimuthFrequency::Float64
+    slantRangeTime::Millisecond
+    incidenceAngleMidSwath::Float64
+    azimuthPixelSpacing::Float64
+    numberOfSamples::Int
+end
+
+"""
+SwathTiming
+
+returns structure of SwathTiming from metadata in .xml
+"""
+Base.@kwdef struct SwathTiming
+    linesPerBurst::Int64
+    samplesPerBurst::Int64
+    burstCount::Int32
+end
+
+"""
+Burst
+
+returns structure of Burst from metadata in .xml
+"""
+Base.@kwdef struct Burst
+    burstNumber::Int32
+    azimuthTime::DateTime
+    sensingTime::DateTime
+    azimuthAnxTime::Millisecond
+    byteOffset::Int64
+    firstValidSample::Vector{Int64}
+    lastValidSample::Vector{Int64}
+    burstId::Int64
+    absoluteBurstId::Int64
+end
+
+
+
+"""
+DopplerCentroid
+
+returns structure of DopplerCentroid from metadata in .xml
+"""
+Base.@kwdef struct DopplerCentroid
+    burstNumber::Int32
```

Comment:
drop burstNumer in DopplerCentroid and make it a field in burst. (Same for AzimuthFmRate)

```
Base.@kwdef struct Burst
    burstNumber::Int32
    azimuthTime::DateTime
    sensingTime::DateTime
    azimuthAnxTime::Millisecond
    byteOffset::Int64
    firstValidSample::Vector{Int64}
    lastValidSample::Vector{Int64}
    burstId::Int64
    absoluteBurstId::Int64
    dopplerCentroid::DopplerCentroid
    azimuthFmRate::AzimuthFmRate
end
```

Reply:
Added both Doppler.. and Azimuth.. to burst structure. The BurstInfo is now a structure of ::Vector{Burst}